src/SPC/builder/unix/UnixBuilderBase.php

```diff
@@ -173,7 +173,11 @@ protected function sanityCheck(int $build_target): void
         }
 
         // sanity check for embed
-        if (($build_target & BUILD_TARGET_EMBED) === BUILD_TARGET_EMBED) {
+        if (($build_target & BUILD_TARGET_EMBED) === BUILD_TARGET_EMBED ||
+            file_exists(BUILD_BIN_PATH . '/php-config') &&
+            file_exists(BUILD_BIN_PATH . '/phpize') &&
+            (file_exists(BUILD_LIB_PATH . '/libphp.a') || file_exists(BUILD_LIB_PATH . '/libphp.so'))
+        ) {
```

Comment:
I feel like we only need to check this when we are building shared extensions. This will result in redundant testing when building the static extension after building the shared extension.

Reply:
You're right, it doesn't make much sense to rebuild a static installation without changing libs/extensions.

Reply:
Ultimately we should only run checks in embed mode.